tests/meta/compress/test_clean_fastq.py

```diff
@@ -205,3 +209,33 @@ def test_cli_clean_fastqs_pending_compression_metadata(
     # THEN assert that the FASTQ files are NOT removed
     assert fastq_first.exists()
     assert fastq_second.exists()
+
+
+def test_clean_selected_fastq_files(
```

Comment:
```suggestion
def test_clean_fastq_files_new_case(
```
or something a bit more descriptive?